### ISSUE TITLE
Add a performance mode

### ISSUE THREAD

**Issue by adaur** [featherbb/featherbb#8]
We'll strip out the Online feature and some more when this mode is enabled

**Comment by axisdos** [comment]
Yes! This is the perfect mode. Please remove all the unnecessary features when this mode is activated. This would improve the performance.
